zulip_bots/zulip_bots/run.py

```diff
@@ -9,13 +9,15 @@
 from types import ModuleType
 from importlib import import_module
 from os.path import basename, splitext
+from typing import Any, Optional, Text
 
 from zulip_bots.lib import run_message_handler_for_bot
 from zulip_bots.provision import provision_bot
 
 current_dir = os.path.dirname(os.path.abspath(__file__))
 
 def import_module_from_source(path, name=None):
+    # type: (Text, Optional[Text]) -> Any
```

Comment:
@neiljp I think this is the only place where I can use `types.ModuleType` right now.